src/core/sys/darwin/sys/event.d

```diff
@@ -43,7 +43,6 @@ enum : short
     EVFILT_FS       =  -9, /* filesystem events */
     EVFILT_USER     = -10, /* User events */
     EVFILT_VM       = -12, /* virtual memory events */
-    EVFILT_SYSCOUNT =  11
```

Comment:
Perhaps we should deprecate this constant first and remove in the next release cycle?